### ISSUE TITLE
Links within `ins` elements cannot be distinguished visually

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/tr-design#325]
Underlining insertions of text using `<INS>` makes it hard to find links that are on the inserted text, as you cannot visibly see any difference in the underline. See the screen shot below for the WCAG2ICT WG Note, where the the exception text has links on "user agent" and "software" but not on "or".
<img width="798" alt="" src="https://user-images.githubusercontent.com/5894100/232610775-702d6976-9bea-4e05-a363-b36fa4eba937.png">

Would it be possible for us to tweak the styling applied for `ins` elements so that there is a clear visual distinction of links within `inst` elements?

**Comment by fantasai** [comment]
We could use a dashed underline style for insertions, would that help?

**Comment by maryjom** [comment]
@fantasai I think that is worth a try for sure.

**Comment by daniel-montalvo** [comment]
Thank you very much.

+1 for that @fantasai

**Comment by fantasai** [comment]
@daniel-montalvo OK, this is now deployed on w3.org; closing out the issue. :) File a new one if there's a problem~

**Comment by maryjom** [comment]
@daniel-montalvo @fantasai I've no idea how this formatting change gets propagated into the WAI documents or if there's something incorrect in the code change, but the dotted underline still isn't present in my recently built file. The text still appears with a solid underline. Here's a link to an example where it's used [definition of target](https://deploy-preview-171--wcag2ict.netlify.app/#dfn-target)
